src/test/java/io/github/genomicdatainfrastructure/discovery/api/DatasetSearchTest.java

```diff
@@ -31,8 +51,15 @@ void can_anonymously_search_datasets() {
     }
 
     @Test
-    void can_search_datasets() {
+    void can_search_datasets_with_beacon_filters() {
         var query = DatasetSearchQuery.builder()
+                .facets(List.of(
+                        DatasetSearchQueryFacet.builder()
+                                .facetGroup("beacon")
```

Comment:
If beacon has its own filters, how will frontend know about these filters?

Reply:
This service is following CKAN `/package_search` approach. That means in the response we have both results and search facets. We introduced a tiny change, the concept of `facetGroup`, just to tell to the backend which source the filter/facet belongs to. CKAN facets will be used as filter only by CKAN module, and `beacon` facets will be used as filter only by Beacon module.

Reply:
There is another class that fetches filtering terms (how facets/filters are called in Beacon).